packages/cli/src/commands/copy.ts

```diff
@@ -0,0 +1,94 @@
+/**
```

Comment:
Could we default to a list of directories based on XDG_DATA_DIRS and print what directories we are searching?

We could construct the shell and mutter directories based on the data directories too.